data/mods/Xedra_Evolved/itemgroups/itemgroups.json

```diff
@@ -186,7 +186,6 @@
       [ "sleeveless_trenchcoat_fur", 10 ],
       [ "sleeveless_duster_fur", 10 ],
       [ "sleeveless_trenchcoat_faux_fur", 10 ],
-      [ "coat_fur_sf", 50 ],
```

Comment:
What is this and why is it getting removed in a pants pr?

Reply:
> adds cosmetic variants for various items, including regular pants, fur coats,

Apart for pants, which is the main element of this PR, I also bundled in variants for fur coats. “coat_fur_sf” was the id for the sable fur coat, which has been iced and made into a variant of the basic fur coat item.